src/RevisionPopupWidget.js

```diff
@@ -36,6 +36,26 @@ function getSizeHtml( size ) {
 		'</span>';
 }
 
+function getCommentHtml( data ) {
```

Comment:
Please add documentation for this method

(Also, we missed it for the other method above, can be good to add there too)

Reply:
Doh! I am religious about this, but apparently just as forgetful, hehe. We should add https://eslint.org/docs/rules/require-jsdoc